Deep-Drop/core/macros/template.vbs

```diff
@@ -1,58 +1,53 @@
-Sub OnOpen()
```

Comment:
vbs/vba/txt just needs copy + pasting into a doc.

Reply:
For some reason Git shows this as if I renamed and modified template.vbs, but that's not the case. The template.vba file is a copy of template.vbs with some modifications to allow executing it via wscript.exe/cscript.exe for those poor souls who would like to test this on a VM without Microsoft Word installed (like me).

Reply:
That's fair, this is obviously in the context of a phishing email - but using this internally to detect a safe pivot or something could be an interesting use case.